frontend/src/components/pages/RegistrantExperience/SessionSelection/SessionCard.tsx

```diff
@@ -27,13 +27,14 @@ const SessionCardDetails = ({
   campSession,
 }: SessionCardDetailsProps): React.ReactElement => {
   const getCampSessionDates = () => {
-    const startDate = new Date(campSession.dates[0]);
-    const endDate = new Date(campSession.dates[campSession.dates.length - 1]);
+    const datesToUse = sortDateStrings(campSession.dates);
+    const startDate = new Date(datesToUse[0]);
+    const endDate = new Date(datesToUse[campSession.dates.length - 1]);
     return `${startDate.toLocaleString("default", {
       month: "short",
-    })} ${startDate.getDay()} - ${endDate.toLocaleString("default", {
```

Comment:
Funny little bug, getDay() returns day of the week, not the day of the month. That's what getDate() is for.